public/resources/js/github-releases.js

```diff
@@ -0,0 +1,70 @@
+class Release {
+    name = '';
+    displayName = '';
+    downloads = [];
+    isGui = false;
+}
+
+async function loadReleases() {
+    const cacheKey = 'vrc-get/releases';
+    // Check if cached result exists and is not expired
+    const cachedResult = JSON.parse(localStorage.getItem(cacheKey));
+    if (cachedResult?.timestamp && Date.now() - cachedResult.timestamp < 1800000) { // 30 minutes = 1800 seconds = 1,800,000 milliseconds
+        console.info('Got releases from cache instead of API.')
+        return cachedResult.data;
+    }
+
+    // If not cached or expired, fetch from GitHub
+    try {
+        let releases = [];
+        let page = 1;
+        while (true) {
+            const response = await fetch(`https://api.github.com/repos/vrc-get/vrc-get/releases?per_page=100&page=${page}`);
```

Comment:
also for this releases API call.

I can make CI to update website on cli release so I want to put some json (or embed data) to retrieve all cli release on website.